### PR TITLE
add 60% pcb for the smk 2nd switch(with mx mount)

### PR DESCRIPTION
**Pull request by yulei**
## Description
add a gh60 compatible pcb with support of the smk 2nd switch.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/smk60/keymaps/60_ansi/keymap.c [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+     * |-----------------------------------------------------------|
+     * |Contro|  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|Enter   |
+     * |-----------------------------------------------------------|
+     * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift |Fn0|
+     * |-----------------------------------------------------------'
+     * |Ctrl |Gui|Alt  |         Space         |Alt  |Gui|Fn |Ctrl |
+     * `-----------------------------------------------------------'
+     */
+    [0] = LAYOUT_60_ansi(
+      KC_GRV,      KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,   KC_MINS,KC_EQL, KC_BSPC,\
```
Comment (fauxpark) [inline keyboards/smk60/keymaps/60_ansi/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".

File: keyboards/smk60/smk60.c [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/**
+ * smk60.c
+ */
+
+#include QMK_KEYBOARD_H
```
Comment (fauxpark) [inline keyboards/smk60/smk60.c]:
```suggestion
#include "smk60.h"
```

File: keyboards/smk60/rules.mk [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+RGBLIGHT_ENABLE = yes       # Use RGB bottom light
+LAYOUTS = 60_ansi 60_hhkb
```
Comment (fauxpark) [inline keyboards/smk60/rules.mk]:
```suggestion
LAYOUTS = 60_ansi 60_hhkb 60_iso
```

File: keyboards/smk60/smk60.c (hunk 2 above)
Comment (yulei) [reply keyboards/smk60/smk60.c]: Is this the current recommend style? last time seems QMK_KEYBOARD_H was preferred.

Comment (fauxpark) [reply keyboards/smk60/smk60.c]: Yes, but not in this file. `#include QMK_KEYBOARD_H` goes in `keymap.c`.

File: keyboards/smk60/keymaps/60_ansi/keymap.c (hunk 1 above)
Comment (yulei) [reply keyboards/smk60/keymaps/60_ansi/keymap.c]: they were removed

File: keyboards/smk60/smk60.c (hunk 2 above)
Comment (yulei) [reply keyboards/smk60/smk60.c]: thanks, they were fixed

**Pull request merged**